<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>475</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>UPS监控系统（IPM4X）</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/icon/image/2.jpeg</normaloff>:/icon/image/2.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>串口选择：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="comboxex" name="mPort_comboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="mRefresh_pushButton">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mOpen_pushButton">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mClear_pushButton">
        <property name="text">
         <string>清屏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mShowdata_pushButton">
        <property name="text">
         <string>实时数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="mDebug_textEdit">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>comboxex</class>
   <extends>QComboBox</extends>
   <header location="global">comboxex.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
